jerry-core/ecma/operations/ecma-exceptions.c

```diff
@@ -414,6 +533,25 @@ ecma_raise_uri_error (const char *msg_p) /**< error message */
   return ecma_raise_standard_error (ECMA_ERROR_URI, (const lit_utf8_byte_t *) msg_p);
 } /* ecma_raise_uri_error */
 
+#if JERRY_BUILTIN_AGGREGATE_ERROR
+
+/**
+ * Raise a AggregateError with the given errors and message.
+ *
+ * @return ecma value
+ *         Returned value must be freed with ecma_free_value
+ */
+ecma_value_t
+ecma_raise_aggregate_error (ecma_value_t errors, ecma_value_t msg) /**< error message */
```

Comment:
Comment is missing for errors, and it should have the same names as `ecma_new_aggregate_error`.